spring-boot-project/spring-boot/src/main/java/org/springframework/boot/web/embedded/jetty/JettyWebServer.java

```diff
@@ -168,8 +168,8 @@ public void start() throws WebServerException {
 					}
 				}
 				this.started = true;
-				logger.info("Jetty started on port(s) " + getActualPortsDescription() + " with context path '"
-						+ getContextPath() + "'");
+				logger.info("Jetty started on port" + (connectors.length == 1 ? "" : "s") + " "
```

Comment:
I think it might be nice if the use of `connectors` was kept encapsulated in `getActualPortsDescription()`. Perhaps that method could be changed to return strings like "port 8080 (http)" or "ports 8080 (http), 8443 (https)"